ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ToolkitConfigurationService } from 'toolkit';
+import { Auth0AdapterService, InvitationServiceAgent } from 'ddp-sdk';
+import { ErrorType } from '../../models/errorType';
+import { take } from 'rxjs/operators';
+
+@Component({
+  selector: 'user-registration-prequal',
+  templateUrl: './user-registration-prequal.component.html',
+  styleUrls: ['./user-registration-prequal.component.scss']
+})
+export class UserRegistrationPrequalComponent implements OnInit {
+  public formGroup: FormGroup;
+  public errorMessage: string | null = null;
+
+  constructor(
+    private auth0: Auth0AdapterService,
+    private invitationService: InvitationServiceAgent,
+    @Inject('toolkit.toolkitConfig') public config: ToolkitConfigurationService) { }
+
+  public ngOnInit(): void {
+    this.initForm();
+  }
```

Comment:
I extracted form initialization from the constructor, primarily, we use constructor only for injecting dependencies.